webrpc.go

```diff
@@ -26,24 +27,45 @@ func ParseSchemaFile(path string) (*schema.WebRPCSchema, error) {
 		return schema.ParseSchemaJSON(json)
 
 	case ".ridl":
-		// Use root FS to allow RIDL file imports from parent directories,
-		// ie. import ../../common.ridl.
-
-		root := "/"
-
-		// Support Windows paths. Currently only supports paths on the same volume.
-		if volume := filepath.VolumeName(absolutePath); volume != "" {
-			root = volume + "/"
+		root, _, err := getRootPath()
+		if err != nil {
+			return nil, fmt.Errorf("get root path: %w", err)
 		}
 
-		path := filepath.ToSlash(absolutePath[len(root):])
+		schema, err := ridl.NewParser(os.DirFS(root), filepath.ToSlash(absolutePath[len(root):])).Parse()
```

Comment:
I don't find this long in-line line more readable compared to the `path`/`rootFS` variables.